ElevenLabs/Packages/com.rest.elevenlabs/Runtime/Voices/VoicesEndpoint.cs

```diff
@@ -55,29 +55,45 @@ public VoiceResponse([JsonProperty("voice_id")] string voiceId)
 
         protected override string Root => "voices";
 
+        /// <summary>
+        /// Gets a list of all available voices for a user, and downloads all their settings.
+        /// </summary>
+        /// <param name="cancellationToken"></param>
+        /// <returns><see cref="IReadOnlyList{T}"/> of <see cref="Voice"/>s.</returns>
+        public Task<IReadOnlyList<Voice>> GetAllVoicesAsync(CancellationToken cancellationToken = default)
+        {
+            return GetAllVoicesAsync(true, cancellationToken);
```

Comment:
Do we want this to be `true` by default?

Reply:
Personally I would either make `false` the default or just remove the overload altogether. I retained the original overload only for compatibility, but if you are okay with removing it completely or marking it obsolete, I am also okay with that.

Reply:
I'm ok with keeping overload. But lets make it false to mirror the prev functionality.

Reply:
Also be sure to copy this change into the .net version as well.